config/cable.yml

```diff
@@ -1,5 +1,8 @@
 development:
-  adapter: async
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
We should probably pull this from here: https://github.com/psu-libraries/scholarsphere/blob/main/lib/scholarsphere/redis_config.rb .  And do this same in the "production" section below.  We don't store REDIS_URL in our environment variables.  We do store REDIS_HOST, and then the url is pieced together in that RedisConfig class.